config/core/test-configs.yaml

```diff
@@ -2000,6 +2011,7 @@ test_configs:
       - baseline
       - baseline-cip-nfs
       - baseline-nfs
+      - chromeos-flash
```

Comment:
Adding it here means the pipeline logic in Jenkins will be automatically scheduling jobs, which is not what we want.  By specifying the `--plan` and `--target` command line arguments to `kci_test`, it's possible to generate a test definition that's not listed in the YAML.